lib/order.rb

```diff
@@ -0,0 +1,25 @@
+require_relative 'menu'
+
+class Order
+
+attr_reader :user_input, :current_order, :meal_quantities, :meal_selection
+
+  def initialize(user_input, menu_choice)
+    @user_input = user_input
+    @current_order = current_order
+    @menu_choice = menu_choice
+    @bill = 0
+  end
+
+
+
+  def process_order
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for process_order is too high. [21.21/15] (http://c2.com/cgi/wiki?AbcMetric)